Biology.java

```diff
@@ -0,0 +1,346 @@
+import cern.colt.Arrays;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * A class that allows antigen to model mutations more realistically, thus, respecting biology
+ *
+ * @author Thien Tran
+ */
+public class Biology {
+    public enum DMSData {
```

Comment:
Could we add a description here, or in `description.md` about the expected file format. For example, some requirements might be:

- DMS data has to have the same number of amino-acid sites as the input sequence
- sites must occur in sequential numbering (the site column in the DMS data is ignored)
- the file has a single header line (giving column names)
- amino acids are assumed to occur in a particular order (right? if so what is this order?)